cmd/ndm_daemonset/probe/mountprobe.go

```diff
@@ -0,0 +1,90 @@
+package probe
```

Comment:
licence header missing here.
@kmova can we have same licence check which is in `openebs/maya`.

Reply:
Added license header to new files